packages/flutter/lib/src/material/outline_button.dart

```diff
@@ -494,10 +521,8 @@ class _OutlineBorder extends ShapeBorder {
 
   @override
   bool operator ==(dynamic other) {
-    if (identical(this, other))
-      return true;
-    if (runtimeType != other.runtimeType)
-      return false;
+    if (identical(this, other)) return true;
+    if (runtimeType != other.runtimeType) return false;
```

Comment:
please leave this as it was before, this is a pattern we have all over the codebase.

Reply:
Sorry that was dartfmt again, reverted